### PR TITLE
Support run multiple games with seperated category

### PR DESCRIPTION
**Pull request by nitro2**
We can have 1 or 2 bot running on separated Category 
```
--GAME                   --> short game
     |-- #lobby
     |-- #gameplay
--GAME_DAILY             --> long game for realtime play
     |-- #lobby
     |-- #gameplay
```
Currently only need to change `GAME_CATEGORY = "GAME_DAILY"` in config.py

NOTE:
- GAME_CATEGORY and its channels (lobby and gameplay) will be created at bot startup.
- Bot only receives command under channels belong to GAME_CATEGORY
- In case GAME_CATEGORY is deleted, admin must restart bot.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -16,9 +16,14 @@ async def process_message(message):
         await command.parse_command(game, message)
 
 
-def verify_ok(user):
-    # TODO: Check valid user in valid channel
-    return True
+def verify_ok(message):
+    try:
+        if message.channel.category.name == config.GAME_CATEGORY:
```
Comment (Stanley00) [inline bot.py:21]: Should be `return message.channel.category.name == config.GAME_CATEGORY`

**Review by Stanley00 — APPROVED** [review]

**Comment by kaizerstudio** [comment]
> We can have 1 or 2 bot running on separated Category
> 
> ```
> --GAME                   --> short game
>      |-- #lobby
>      |-- #gameplay
> --GAME_DAILY             --> long game for realtime play
>      |-- #lobby
>      |-- #gameplay
> ```
> 
> Currently only need to change `GAME_CATEGORY = "GAME_DAILY"` in config.py
> 
> NOTE:
> 
>     * GAME_CATEGORY and its channels (lobby and gameplay) will be created at bot startup.
> 
>     * Bot only receives command under channels belong to GAME_CATEGORY
> 
>     * In case GAME_CATEGORY is deleted, admin must restart bot.

I have a question. There are only 2 categories, or admin can create custom categories whenever they want?

**Comment by nitro2** [comment]
> I have a question. There are only 2 categories, or admin can create custom categories whenever they want?

Currently we cannot create 2 categories on same bot. We use fixed `config.GAME_CATEGORY` . 
Question: What behavior in this pull request?
 - Bot starts up and checks whether the `config.GAME_CATEGORY` exists. If not, it creates one.
 
We may support multiple categories on same bot in near future.

File: bot.py (hunk 1 above)
Comment (nitro2) [reply bot.py:21]: message.channel.category.name can be None. Does it work?

**Pull request merged**